packages/slate-react/src/components/content.js

```diff
@@ -9,7 +9,6 @@ import {
   SUPPORTED_EVENTS,
 } from 'slate-dev-environment'
 import logger from 'slate-dev-logger'
-import throttle from 'lodash/throttle'
```

Comment:
I don't know about the specific fo this PR and if it's the right solution, I'll let @ianstormtaylor reviews it.

But if we remove this include, we can remove `lodash/throttle` from the dependencies in `package.json` (I'm always happy to see lodash being removed from dependencies 😍).

Reply:
@SamyPesse Looks like we're including lodash as a single package, and not as individual pieces I can remove, unfortunately.